BDArmory/VesselSpawning/SpawnUtils.cs

```diff
@@ -1142,17 +1142,17 @@ public void ApplyFJRT(Vessel vessel)
                 var AI = VesselModuleRegistry.GetModule<BDModulePilotAI>(vessel);
                 if (AI != null)
                 {
-                    AI.extendTargetDist = Mathf.Min(AI.extendTargetDist, BDArmorySettings.PS_EXTEND_DIST);
-                    AI.collisionAvoidanceThreshold = Mathf.Max(AI.collisionAvoidanceThreshold, BDArmorySettings.PS_AVOID_THRESH);
-                    AI.vesselCollisionAvoidanceLookAheadPeriod = Mathf.Max(AI.vesselCollisionAvoidanceLookAheadPeriod, BDArmorySettings.PS_AVOID_LA);
-                    AI.vesselCollisionAvoidanceStrength = Mathf.Max(AI.vesselCollisionAvoidanceStrength, BDArmorySettings.PS_AVOID_STR);
-                    AI.idleSpeed = Mathf.Max(AI.idleSpeed, BDArmorySettings.PS_IDLE_SPEED);
+                    if (BDArmorySettings.PS_EXTEND_DIST > 0) AI.extendTargetDist = Mathf.Min(AI.extendTargetDist, BDArmorySettings.PS_EXTEND_DIST);
```

Comment:
Collision Avoidance Threshold, CA Look Ahead, and CA Strength could conceivably be 0.
Technically so to could be Extend Distance, Cockpit view distance and guardAngle, but those would cause problems if set to 0.
None of them can or should be negative, which is why -1 is the current 'don't use' default for the values.
Wouldn't it be simpler to just do >= 0 for the Collision Avoidance settings?

Reply:
Yeah, I didn't look at the default values when I wrote that. I think as long as the `VAR > X` or `VAR >= X` condition is such that all valid values and only valid values for the override can be used, then it should be fine.